ArduCopter/mode_guided.cpp

```diff
@@ -522,8 +522,8 @@ bool ModeGuided::set_destination(const Location& dest_loc, bool use_yaw, float y
     return true;
 }
 
-// set_velaccel - sets guided mode's target velocity and acceleration
-void ModeGuided::set_accel(const Vector3f& acceleration, bool use_yaw, float yaw_rad, bool use_yaw_rate, float yaw_rate_rads, bool relative_yaw, bool log_request)
+// set_vel_accel_neu_cm - sets guided mode's target velocity and acceleration
+void ModeGuided::set_accel_neu_cmss(const Vector3f& accel_neu_cm, bool use_yaw, float yaw_rad, bool use_yaw_rate, float yaw_rate_rads, bool relative_yaw, bool log_request)
```

Comment:
similar to the comments in the .h file, I think the argument names for the next few function definitions should be updated to:

- vel_neu_cm to vel_neu_cms
- accel_neu_cm to accel_neu_cmss